subprojects/packagefiles/spdk/meson.build

```diff
@@ -20,14 +20,28 @@ if get_option('build_subprojects') and not fs.exists('build' / 'lib' / 'src' / '
     check: true,
   ).stdout().strip()
 
+  message('Checking if generic platform (Apple Silicon)')
+  if target_machine.cpu_family() == 'aarch64'
```

Comment:
How about:
```
is_platform_generic = target_machine.cpu_family() == 'aarch64' and run_command([
    import('python').find_installation('python3'),
    '-c',
    'print(open("/sys/devices/system/cpu/cpu0/regs/identification/midr_el1").read() == "")'
  ],
  check: true,
).stdout().strip() == 'True'
```

?

Reply:
You're right. I originally intended to return the string, that's why. Fixed now.